src/Mod/PartDesign/Gui/TaskFilletParameters.cpp

```diff
@@ -195,6 +207,54 @@ void TaskFilletParameters::apply()
     }
 }
 
+void TaskFilletParameters::setupGizmos(ViewProviderDressUp* vp)
+{
+    if (!Gizmos::isEnabled()) {
+        return;
+    }
+
+    gizmos = std::make_unique<Gizmos>();
+
+    auto radiusGizmo = new Gui::LinearGizmo(ui->filletRadius);
+    auto radiusGizmo2 = new Gui::LinearGizmo(ui->filletRadius);
+
+    gizmos->addGizmo(radiusGizmo);
+    gizmos->addGizmo(radiusGizmo2);
+    gizmos->initGizmos();
+
+    setGizmoPositions();
+
+    vp->attachGizmos(gizmos.get());
+}
+
+void TaskFilletParameters::setGizmoPositions()
+{
+    if (!gizmos) {
+        return;
+    }
+
+    auto fillet = getObject<PartDesign::Fillet>();
+    Part::TopoShape baseShape = fillet->getBaseTopoShape();
+    std::vector<Part::TopoShape> shapes = fillet->getContinuousEdges(baseShape);
+
+    if (shapes.size() == 0) {
+        gizmos->visible = false;
+        return;
+    } else {
+        gizmos->visible = true;
+    }

```

Comment:
Same thing here if visible defaults to true (which it should)
```suggestion
    if (shapes.size() == 0) {
        gizmos->visible = false;
        return;
    }
```

Reply:
Should be fixed.